src/config.js

```diff
@@ -11,8 +26,16 @@ if (process.env.NODE_ENV === 'test') {
 	})}
 
 
+const { error, value: envVars } = environmentSchema.prefs({
+	errors: { label: 'key' }
```

Comment:
not a review, just curious as to what this does

Reply:
> not a review, just curious as to what this does

To be honest, I don't fully understand it myself but I saw it in the documentation when I was researching and decided to use  it.

Reply:
I have taken it out though @jocrah

Reply:
> To be honest, I don't fully understand it myself but I saw it in the documentation when I was researching and decided to use it. @jocrah

okay understood :), wanted to learn from it myself hence the question

Reply:
> I have taken it out though @jocrah

o okay, alright, was trying to check it out myself but not arrived at an understanding yet. maybe later then

Reply:
@jocrah https://joi.dev/api/?v=17.6.1#checkpreferencesprefs

Reply:
cool, thanks @itzgeebee, will check it out some more soon